manifest/x86_64/h/highway.filelist

```diff
@@ -1,6 +1,7 @@
-# Total size: 9263402
+# Total size: 26818890
```

Comment:
I think there is something wrong with the calculation.  This went way up and the size varies greatly between architectures when it should be relatively the same.